src/framework/uicomponents/qml/MuseScore/UiComponents/GridViewSectional.qml

```diff
@@ -24,6 +25,9 @@ Item {
     property int columns: noLimit
     property int columnSpacing: 2
 
+    property int spacingBeforeSection: isHorizontal ? columnSpacing : rowSpacing
```

Comment:
better move it to private properties

Reply:
I put it here deliberately, thinking we might want to override it at some point. Shall I move it to private for now? If we somewhen need it, we can move it back.

Reply:
The second sentence is correct :)